app/javascript/channels/collect_submissions_channel.js

```diff
@@ -0,0 +1,24 @@
+import consumer from "./consumer";
+
+function create_collect_submissions_channel_subscription() {
```

Comment:
Great! I couldn't see where this function was being called, but I think the right place to do it is in the `SubmissionTable` React component. If you do it in the `componentDidMount` method it should make the table automatically listen for updates (even from another tab).